app/scripts/templates/settings/clients.mustache

```diff
@@ -75,7 +75,7 @@
       </ul>
 
       <div class="button-row">
-        <button type="submit" class="settings-button primary clients-refresh">{{#t}}Refresh{{/t}}</button>
+        <button type="submit" class="settings-button primary clients-refresh" data-min-progress-indicator-ms="1800">{{#t}}Refresh{{/t}}</button>
```

Comment:
Original issue mentioned matching hamburger sync spinner delay which is 1600ms. @ryanfeeley should this be 1600 or 1800ms?